.github/workflows/ci.yml

```diff
@@ -1,5 +1,7 @@
 on:
   push:
+    branches: [master]
+  pull_request:
```

Comment:
Does this work as intended? It seems that current PRs don't get CI runs. Maybe:
```yaml
  push:
    branches: [ "master" ]
  pull_request:
    branches: [ "master" ]
```

Reply:
#134